README.md

```diff
@@ -50,28 +50,157 @@ const nearRpcProvider = new NearRpcProvider('near')
 
 ## Methods
 
-### Block number
+### Block Number
 
 Returns the block number of the latest block
 
 ```typescript
 const blockNumber = await nearRpcProvider.getBlockNumber()
 ```
 
-### Get an account balance
+### Call a Contract Function
+
+Allows you to call a contract method as a view function.
+
+```typescript
+const contractResponse = await nearRpcProvider.contractCall(
+  'example.testnet', 'latest', 'getMessages', 'e30='
+)
+```
+
+### Get an Account Balance
 
 Returns a BigNumber representing the balance on the provided account
 
 ```typescript
-const balance = await nearRpcProvider.getBalance('blockcoders.testnet')
+const balance = await nearRpcProvider.getBalance('example.testnet')
+```
+
+### Get an Access Key
+
+Returns information about a single access key for given account.
+
+```typescript
+const accessKey = await nearRpcProvider.getAccessKey(
+  'example.testnet',
+  'ed25519:H9k5eiU4xXS3M4z8HzKJSLaZdqGdGwBG49o7orNC4eZW',
+  75866664,
+)
+```
+
+### Get an Access Key List
+
+Returns all access keys for a given account. You can querying it by `finality`:
+
+```typescript
+const accessKeyList = await nearRpcProvider.getAccessKeyList('example.testnet', 'latest')
+```
+
+By `block_height`:
+
+```typescript
+const accessKeyList = await nearRpcProvider.getAccessKeyList('example.testnet', 27912554)
+```
+
+Or by `block_hash`:
+
+```typescript
+const accessKeyList = await nearRpcProvider.getAccessKeyList(
+  'example.testnet', '3Xz2wM9rigMXzA2c5vgCP8wTgFBaePucgUmVYPkMqhRL'
+)
 ```
 
-### Get block
+### Get Block
 
-Returns a [block](https://docs.ethers.io/v5/api/providers/types/#providers-Block) by `block_id` or `block_hash`
+[DEPRECATED] Throws the following erorr:
```

Comment:
I will say here "Use Get Block With Chunk instead of"

Reply:
Got this. Fixed!